controllers/reconcilers/kserve_networkpolicy_reconciler.go

```diff
@@ -30,14 +31,19 @@ import (
 )
 
 const (
-	networkPolicyName = "allow-from-openshift-monitoring-ns"
+	monitoringNetworkPolicyName            = "allow-from-openshift-monitoring-ns"
```

Comment:
```suggestion
	uwmNWPName                           = "allow-from-uwm-ns"
	openshiftIngressNWPName      = "allow-from-openshift-ingress-ns"
	opendatahubOwnedNWPName            = "allow-from-opendatahub-ns"
```

Reply:
I can guess what `uwm` stands for but would like to know the reason for this change :) 

Important to note is that it's already in use, so renaming it would result in adding new policy under "uwm" name with the same rules.

Reply:
The net policy defined later on has a ns selector `openshift-user-workload-monitoring"` and that means it applies to uwm so people reading this name will expect to apply for openshift-monitoring ns. It is more about consistency I think.

Reply:
if i recall it correctly, from 2.5 release, serving started to use `user-workload-monitoring` namespace for shipping their metrics to. pre-2.5 it was the standard `openshift-monitoring` namespace. 
also I remember there is a ticket on serving team to "cleanup the old monitoring stack" which should remove resource created in old releases, including this "allow-from-openshift-monitoring-ns" NWP  ( the one created by modelmesh not by knatvie)

Reply:
> if i recall it correctly, from 2.5 release, serving started to use user-workload-monitoring namespace for shipping their metrics to

Not really we haven't changed anything. 

> also I remember there is a ticket on serving team to "cleanup the old monitoring stack" which should remove resource created in old releases, including this "allow-from-openshift-monitoring-ns" NWP ( the one created by modelmesh not by knatvie)

Have a pointer?

Reply:
@danielezonca you remember any jira was created to do the cleanup work from old monitoring stack?  i can only find https://issues.redhat.com/browse/RHOAIENG-293 but from the last linked PR, does not like it has touched NWP

Reply:
I think @VedantMahabaleshwarkar is still working on that.
@VedantMahabaleshwarkar Please have a look here to make sure we align the changes

Reply:
Right. He is still working on this.
This is [@VedantMahabaleshwarkar PR](https://github.com/opendatahub-io/odh-model-controller/pull/146). @vaibhavjainwiz and I think using init-container to remove old monitoring stack is not a good idea. So 
 @VedantMahabaleshwarkar and @vaibhavjainwiz are discussing how to implement to remove old monitoring stack.

Reply:
@Jooho @zdtsw @danielezonca My PR for https://issues.redhat.com/browse/RHOAIENG-293 is to remove the lingering resources related to the monitoring stack. i. e the Prometheus operator deployments and operator pods. It does not touch Networkpolicies as the old monitoring stack itself did not create any Network Policies

Reply:
The `allow-from-openshift-monitoring-ns` NetworkPolicy was added for Kserve during the work to enable Kserve metrics in UWM so I have not touched that in my PR

Reply:
Commenting about the original suggested change, I don't think it is worth to do the renaming of the NP. In existing releases, the NP is already created with `allow-from-openshift-monitoring-ns` name. Since we want to fully move away from creating NPs, spending effort on a rename (i.e. migrating from `allow-from-openshift-monitoring-ns` -> to `allow-from-uwm-ns`) is probably not worth the time.